2018AsianGamesApp/app/src/main/java/com/example/mrtbkwjn/a2018asiangamesapp/StreamingActivity.java

```diff
@@ -0,0 +1,25 @@
+package com.example.mrtbkwjn.a2018asiangamesapp;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+
+import com.example.mrtbkwjn.a2018asiangamesapp.liveVideoPlayer.LiveVideoPlayerActivity;
+
+
+public class StreamingActivity extends AppCompatActivity {
+
+    public final static String RTMP_BASE_URL = "//rtmp://192.168.1.104/LiveApp/";
```

Comment:
This IP looks local (192.168.1.xxx is usually used for local wifi) please remember to switch it to the correct one

Reply:
well.. it actually working using my Bolt Mi-Fi.. and i think i'll use it for the final presentation..

Reply:
its looks local because i merge my PC and the virtualBox IP..